data/wanderer/wanderers start.txt

```diff
@@ -260,113 +262,67 @@ mission "Wanderers: Translation Machine"
-	landing
-	source "Greenwater"
-	destination "Alexandria"
-	to offer
-		has "Wanderers: Translation Machine: done"
-		not "human cultural data"
-	on offer
-		conversation "eruk cultural data"
-	on visit
-		dialog
-			`The station has a small museum gift shop that sells a copy of the entire archive on a data card. It only costs 40 credits, but because you have done a horrible job at managing your finances you do not have even that much cash on hand right now.`
-			`	Go earn some money, then return here.`
 	on complete
 		set "human cultural data"
 		payment -40
+		conversation
+			`<Library search> She directs you to the museum gift shop, where, to your surprise, you could buy the full "Human Cultural Archive" for a mere 40 credits.`
+				to display
+					not "Bankrupt at Alexandria"
+			`Now that you finally have 40 credits, you purchase the "Human Cultural Archive" that you must take to <planet>.`
+				to display
+					has "Bankrupt at Alexandria"
+	on visit
+		set "Bankrupt at Alexandria"
+		conversation
+			`<Library search> She directs you to the museum gift shop, to buy the "Human Cultural Archive" for 40 credits.`
+			`	Unfortunately, you have done such a horrible job at managing your finances you do not have even that much cash on hand right now.`
+			`	Go earn some money, then return here.`
 
 
 
-mission "Cultural Data to Greenwater"
-	description "Bring a copy of the human cultural archives to <destination> in exchange for a translation machine."
+mission "Wanderers: Algorithm Training Data"
+	description "Bring a copy of the human cultural archives to <destination> so the Wanderers can train an automated translation algorithm."
 	landing
 	source "Alexandria"
-	destination "Greenwater"
+	destination "Vara K'chrai"
 	to offer
-		has "Human Cultural Archives: done"
-	on offer
+		has "Wanderers: Translation Machine: done"
+	on complete
+		event "wanderer translator available" 56
 		conversation
-			`You had wondered how you would get a copy of the entire cultural archive, but it turns out to be easier than expected. The station has a small museum gift shop, which sells a copy of the archives on a data card. ("The perfect gift for the budding historian in your family! Hold all of human history in the palm of your hand. On sale now for only 39.99!") Given that a cheap data card costs a fraction of a credit, it's a bit of a rip-off, but you gladly buy one to take back to Eruk.`
-				accept
+			`A Wanderer meets you after you land and you hand them the data card. They plug it into a mango-sized machine which seems to adjust its data card slot to match the shape of the card on the way in. The Wanderer makes clicking and whistling noises while manipulating the device. Eventually, the device spits out the phrases, "[Enter, bring] three and five. Within [truth, lies], the pleasant [cow, nova] tickles me [thoughtfully, gracefully]."`
```

Comment:
This is a little bit excessively deus ex machina. As advanced as the wanderers are, a handheld device for machine learning translation on a data set like that feels a little much, though perhaps less so if it's only designed to make a basic assessment of data suitability, and gets combined with a possible need to return and get a more refined data set.

Reply:
I will adjust the story so Sayari had given them a data card reader that they adapted to their technology.

A handheld device is rarely more than an interface: your phone does almost nothing; it is backed by cloud computing. Whatever it is the Wanderer is holding, it is likely backed by a much larger computing center elsewhere that is translating in real-time. With the vast quantity of data Captain Last provided, translating a short phrase is feasible in twenty minutes of back-and-forth.

Really, apart from the mango data-card, this a very most real-world-realistic part of the mission.

Reply:
Captain Last, of course, would not know the mango is backed by a cloud computing center. The story cannot outright state that. However, given that our own society now works off of hand-held devices backed by cloud computing centers, it should be a reasonable assumption to most users. (Or do people think their phones really do have maps of all streets in the world and have the computing capacity to chart a route from Paris to Beijing by bike?)

Reply:
It is now a "fat data pad with several different types of data card slots."

There is nothing "deus ex machina" about a device with a connection to cloud services, so I so no need for other changes.